src/index.js

```diff
@@ -14,13 +14,13 @@ module.exports = class Layer {
 
     var child = this._render()
 
-    return React.render(child, this._mountPoint, cb);
+    return ReactDOM.render(child, this._mountPoint, cb);
```

Comment:
Ah, thanks for the tip. I'm trying to update this now, but I realized that the first argument to `renderSubtreeIntoContainer` is supposed to be the conceptual parent element of the element you're rendering, ie. presumably the one from which context will be passed. It seems like currently there is no way for `Layer` to have a reference to this parent. Should I add `parent` as a third, optional argument to the `Layer` constructor? Then I could use `renderSubtreeIntoContainer` if `parent` is provided, or just `render` otherwise.